TrackletAlgorithm/Constants.h

```diff
@@ -130,7 +131,7 @@ namespace trklet{
   constexpr int N_DISK = 5; // # of endcap disks assumed
 }
 #endif
-constexpr double bfield = 3.8112; // T
+constexpr double bfield = 3.8112; // 
```

Comment:
@aryd Please restore the unit in the comment. We can even write it out, if that makes it clearer.
```suggestion
constexpr double bfield = 3.8112; // Tesla
```

Reply:
Not sure why this would have been removed... Restored.